src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java

```diff
@@ -135,7 +136,12 @@ public FeedLoadResult exportTables() {
                     for (Calendar cal : calendars) {
                         Service service = new Service(cal.service_id);
                         service.calendar = cal;
-                        for (ScheduleException ex : exceptions) {
+                        List<ScheduleException> calendarExceptions = exceptions.stream().filter(ex ->
+                            ex.addedService.contains(cal.service_id) ||
+                            ex.removedService.contains(cal.service_id) ||
+                            ex.customSchedule.contains(cal.service_id)
+                        ).collect(Collectors.toList());
+                        for (ScheduleException ex : calendarExceptions) {
```

Comment:
You can probably use a `.forEach` on this stream to avoid the collector or at least inline this!

Reply:
Makes sense

Reply:
Can we put the stream inside the for statement instead of assigning to a variable?

Reply:
Sure, that's done in https://github.com/conveyal/gtfs-lib/pull/387/commits/e51456e60c8761f646a7bcb71e600676c49606aa although it makes the code maybe a bit harder to read

Reply:
That's not what I meant I meant instead of saving the collected stream to a variable, put the entire stream inside the for definition

Reply:
See inline filtering now please